d3m_profiler/embed.py

```diff
@@ -16,12 +15,10 @@
 tuple(model, emb_size): Tuple(sent2vec.Sent2vecModel, int)
     The Sent2vecModel model and the embedding size.
 """
-def initialize_model(model_weights_path: str) -> (sent2vec.Sent2vecModel, int):
-    model = sent2vec.Sent2vecModel()
-    model.load_model(model_weights_path)
-    emb_size = model.get_emb_size()
-    
-    return model, emb_size
+def initialize_model(model_weights_path: str):
+    model = SentenceTransformer(model_weights_path) 
+       
+    return model
```

Comment:
If this is all its doing, we probably don't need a separate function for initializing the language model.

Reply:
I don't see any changes for this. Is there a reason you like to have this function to wrap one line of code?

Reply:
My embedding is done in the save_embed.py file. I also did not do it in the actual function call because Sentence Transformers actually take a long time to do it. So, I pre-saved the embeddings using save_embed.py.